pkgs/applications/networking/browsers/qutebrowser/default.nix

```diff
@@ -100,10 +77,6 @@ buildPythonApplication {
   dontWrapGApps = true;
   dontWrapQtApps = true;
 
-  preConfigure = lib.optionalString isQt6 ''
```

Comment:
Was this causing an issue? This prevents the man page from building and being included in the final package, so we should probably just add this back if not.